pythonforandroid/recipe.py

```diff
@@ -771,6 +771,7 @@ def should_build(self, arch):
 
         for lib in self.generated_libraries:
             if not exists(join(lib_dir, lib)):
+                print("** Missing library", join(lib_dir, lib))
```

Comment:
Could we log this rather than print?

Reply:
Sure.  I wasn't sure how the log system worked.